docs/book/troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting
+
+vSphere CSI Driver supports 2 levels of logging
+
+- PRODUCTION where all INFO/ERROR logs are logged
+- DEVELOPMENT where all INFO/ERROR/DEBUG logs are logged
+
+The default log level for all the CSI flavors is PRODUCTION. To be able to examine problems related to your environment, modify the log level for the vSphere CSI driver from PRODUCTION to DEVELOPMENT.
+
+**NOTE**: PRODUCTION/DEVELOPMENT log level is available in 2.0 release. For prior releases, use the argument "--v" to change the log level.
+
+## Procedure to change log level
+
+### Vanilla CSI Driver
+
+- To examine the vSphere CSI controller related issues, update the log level for the vsphere-csi-controller and vsphere-syncer containers.
+  - Open the YAML file used for the deployment of vSphere CSI controller and find the LOGGER_LEVEL option in the container's arguments.
+  - Update the logger for this option from PRODUCTION to DEVELOPMENT which will give debug logs too.
+  - Run the kubectl apply command to reload the pods with new configuration.
+
+    ``` sh
+    kubectl apply -f vsphere-csi-controller-deployment.yaml
+    ```
+
+- Also update the log level for vsphere-csi-node daemon set
+  - Open the YAML file used for the vSphere CSI daemonset and find the LOGGER_LEVEL option in the container's arguments.
+  - Update the logger for this option from PRODUCTION to DEVELOPMENT which will give debug logs too.
+  - Run the kubectl apply command to reload the daemonset with new configuration.
+
+    ``` sh
+    kubectl apply -f vsphere-csi-node-ds.yaml
+    ```
+
+### CNS-CSI driver
```

Comment:
naming `CNS-CSI` and `Vanilla CSI Driver` is not proper.

`Vanilla CSI Driver` is also using CNS APIs. Can we name appropriately?

Reply:
We are using the code name CNS CSI for CSI driver deployed on Project Pacific cluster.